front/pages/api/apps/[user]/[sId]/datasets/index.ts

```diff
@@ -68,8 +86,8 @@ export default async function handler(req, res) {
         attributes: ["id", "name", "description"],
       });
 
-      res.status(200).json({ datasets });
-      break;
+      res.status(200).json({ datasets } as GetDatasetResponseBody);
+      return;
```

Comment:
Why switching from breaking to returning here?
I don't especially care about it, I just think I saw in another file where the code after the switch/case/break was actually important. Can't recall where exactly.

Reply:
> I just think I saw in another file where the code after the switch/case/break was actually important

Actually it's why I changed to returns. When break and return are equivalent, I feel return is less ambiguous (especially when looking at a diff, where you don't necessarily see the whole story)

Reply:
definitely not strong opinion here though, I am fine with using break